packages/core/src/babelPlugins/babel-plugin-redwood-cell.ts

```diff
@@ -65,24 +65,24 @@ export default function ({ types: t }: { types: typeof types }): PluginObj {
           }
 
           // Insert at the top of the file:
-          // + import { withCell } from '@redwoodjs/web'
+          // + import { createCell } from '@redwoodjs/web'
           path.node.body.unshift(
             t.importDeclaration(
               [
                 t.importSpecifier(
-                  t.identifier('withCell'),
-                  t.identifier('withCell')
+                  t.identifier('createCell'),
+                  t.identifier('createCell')
                 ),
               ],
               t.stringLiteral('@redwoodjs/web')
             )
           )
 
           // Insert at the bottom of the file:
-          // + export default withCell({ QUERY?, Loading?, Succes?, Failure?, Empty?, beforeQuery?, afterQuery? })
+          // + export default createCell({ QUERY?, Loading?, Succes?, Failure?, Empty?, beforeQuery?, afterQuery? })
           path.node.body.push(
```

Comment:
Should we have a check here if `createCell` is already declared?

Reply:
We already abort the automatic Cell creation if there's a default export